File: internal/ui/commands/skills.go
package commands

func registerSkills(r *Registry) {
	r.Register(&Command{Name: "skills", Description: "List available skills", Kind: Handler,
		Handler: func(args []string, ctx Context) bool {
			if ctx.ListSkillsFn != nil {
				ctx.ListSkillsFn()
			}
			return true
		},
	})
}

File: src/ui/commands/builtin/skills.ts
import type { CommandHandler } from "../registry.js";

export const skillsCommand: CommandHandler = {
  name: "skills",
  description: "List available skills",
  handler: async (_args, ctx): Promise<void> => {
    const skills = ctx.skills.getAvailable();
    // Data only — how a status renders is the renderer's business, and a
    // React element would be dead weight in the service-layer event bus.
    const content =
      skills.length === 0
        ? "(No skills available)"
        : [
            "Available skills:",
            ...skills.map((s) => `  /${s.name} - ${s.description}`),
          ].join("\n");
    ctx.sessionManager.reportStatus({
      role: "status",
      content,
      timestamp: new Date(),
    });
  },
};
